addons/website_forum/tests/test_controller.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+from odoo.tests.common import HttpCase, new_test_user
+
+
+class TestController(HttpCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        portal_user = new_test_user(cls.env, login='portal_user', groups='base.group_portal')
+        cls.portal = portal_user.login
+        admin_user = new_test_user(cls.env, login='admin_user', groups='base.group_user,base.group_system')
+        cls.admin = admin_user.login
+
+        cls.headers = {"Content-Type": "application/json"}
+
+        cls.error = "odoo.exceptions.AccessError"
+
+    def _build_payload(self, params=None):
+        """
+        Helper to properly build jsonrpc payload
+        """
+        return {
+            "jsonrpc": "2.0",
+            "method": "call",
+            "id": 0,
+            "params": params or {},
+        }
+
+    def test_portal_fetch_attachment(self):
+        self.authenticate(self.portal, self.portal)
+        payload = self._build_payload({"res_model": "forum.post"})
+        response = self.url_open('/web_editor/attachment/fetch', data=json.dumps(payload), headers=self.headers)
+        self.assertEqual(200, response.status_code)
+        self.assertNotIn(self.error, response.text)
```

Comment:
Hmmm, this seems a bit weak, if the framework/orm somehow change that one day, that line won't do anything even if the access error is introduced back.

Can't we just check that we have the expected result rather than check we don't have an error (based on string matching)?

Also, if there is an access error on that url_open call, doesn't it makes the test red (and so this line useless)?